lib/libvcc/vcc_backend.c

```diff
@@ -321,6 +321,16 @@ vcc_ParseHostDef(struct vcc *tl, const struct token *t_be, const char *vgcname)
 	    "?proxy_header",
 	    NULL);
 
+	if (tl->t->tok == ID && (vcc_IdIs(tl->t, "none") || vcc_IdIs(tl->t, "None"))) {
```

Comment:
Shouldn't it be a case-insensitive varnish id instead? Is there another occurrence of a keyword having two specific capitalizations?